### ISSUE TITLE
Empty value in the filter should be the first one, not the last one

### ISSUE THREAD

**Issue by tomas-knap** [mff-uk/odcs#1111]
![screen shot 2014-01-25 at 7 11 21 pm](https://f.cloud.github.com/assets/3014917/2002300/3aaa00d2-85ec-11e3-8d43-9ebd16984948.png)

**Comment by tomas-knap** [comment]
Tohle prosim zkontroluj u vsech combo boxu a oprav.

**Comment by kukharm** [comment]
@bogo777 please help wit this issue, i can't find where can i set the order of empty values in the filter.  The problem is in columns with boolean value (also in Sheduler table Status column)

**Comment by bogo777** [comment]
Yes, it cannot be set directly, as combobox filter is generated by addon. We would have to override default component for filter and filter generation, as it is done for "scheduled" column in execution list...

**Comment by tomas-knap** [comment]
Bohuslave, can you please point me to the lines with custom filter?

**Comment by tomas-knap** [comment]
@bogo777
